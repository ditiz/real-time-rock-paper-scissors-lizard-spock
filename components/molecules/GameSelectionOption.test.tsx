import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import GameSelectionOptionMolecule from "./GameSelectionOption"

it("renders GameSelectionOptionMolecule unchanged", () => {
  const { container } = render(<GameSelectionOptionMolecule />)
  expect(container).toMatchSnapshot()
})
