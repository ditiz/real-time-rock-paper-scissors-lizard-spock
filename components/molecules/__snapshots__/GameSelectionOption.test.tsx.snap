// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GameSelectionOptionMolecule unchanged 1`] = `
<div>
  <article
    class="card"
  >
    <a
      href="/test"
    >
      <div>
        <h2>
          test
        </h2>
      </div>
    </a>
  </article>
</div>
`;
